### ISSUE TITLE
Server wide banners

### ISSUE THREAD

**Issue by persalmi** [microsoft/banner-settings-ado-extension#4]
We run some on-premise ADO Servers with many collections. Today we use the banner extension to inform users about events like upcoming service, tips and other stuff.
When doing this on a server with 20+ collections and you need to publish the same banner in all collections it would be very nice to have an option to publish the banner once for all collections in the server.

**Comment by j-windsor** [comment]
Great suggstion! I think this is pretty low priority for us since the vast majority of our customers are Azure DevOps Services users, but if you would like to implement this I will definitely approve the changes.

**Comment by Glenn-Gidlof** [comment]
I agree with the original poster, it would be great to be able to publish to all collections at once.
Or if the banner setting could be programed by API and it could be scripted to push to each or all collections.
